app/controllers/links_controller.rb

```diff
@@ -0,0 +1,43 @@
+class LinksController < ApplicationController
+  def edit
+    @local_authority = LocalAuthority.find_by(slug: params[:local_authority_slug])
+    @service = Service.find_by(slug: params[:service_slug])
+    @interaction = Interaction.find_by(slug: params[:interaction_slug])
+
+    @link = Link.get_link(params)
+    if @link.nil?
+      @link = Link.new
+      @link.url = 'n/a'
```

Comment:
I'm wondering whether we should leave `@link.url` as `nil` here and move the logic for displaying 'n/a' to the view or a helper like LinksHelper? To me showing 'n/a' is more a question of what we display when we get a `nil` link. Changing the value of `@link.url` to 'n/a' technically means that it's not `nil` anymore. Not sure if I'm making sense. What do you think?